### PR TITLE
LUT-27635 : JTA Transactions management and synchronization

### PR DESCRIPTION
**Pull request by evdngsl**
More info https://dev.lutece.paris.fr/gitlab/archi/comite-achitecture/-/issues/52

### CODE REVIEW

File: src/java/META-INF/beans.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<beans xmlns="https://jakarta.ee/xml/ns/jakartaee" 
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+       xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd" 
+       version="4.0" bean-discovery-mode="annotated">
+       
+       <alternatives>
```
Comment (ryahiaoui) [inline src/java/META-INF/beans.xml]: Please remove the commented tags. The commented tags  are related to lutece-core.

**Review by ryahiaoui — COMMENTED** [review]
Please remove the commented tags. The commented tags are related to lutece-core.

**Comment by evdngsl** [comment]
It's done.

**Comment by ryahiaoui** [comment]
thanks

**Pull request merged**